modules/ROOT/pages/to-change-raml-version.adoc

```diff
@@ -20,6 +20,8 @@ you add a new version to an existing asset.
 
```

Comment:
From compacted text:

Each API asset contains a field for the *Version*, which is the version number of the asset, and the *API version*, which identifies the version of the API specification. --SUGGEST-- Each API asset contains a Version field, which is the..., and an API version field, which....


For RAML, OAS, and SOAP assets, the API specification version is coded inside the specification itself. --OK--

Because an HTTP API asset is only an endpoint to which you can provide policies or proxies, the API version field doesn't apply. --SUGGEST-- Because an HTTP API asset is an endpoint, to which you can provide only policies or proxies, the API version field doesn't apply.

The Version and API Version fields are only visible when you add a new asset or when
you add a new version to an existing asset.  --The Version and API version fields are visible only when you add....

[Is the second field "API version" or "API Version"?]

Reply:
Much better! Updated with these changes.

In the UI it's "API version".